### ISSUE TITLE
abstract static member invocation not supported in JS

### ISSUE THREAD

**Issue by DunetsNM** [fable-compiler/Fable#3822]
### Description

Trying to use F# 8's static interfaces,  F# compilation stage passes but then fable errors out unable to transpile it to JS

### Repro code

```fs
type AbstractId =
    abstract member IdStr: string

type AbstractId<'ImplId when 'ImplId :> AbstractId<'ImplId>> =
    inherit AbstractId
    static abstract member TryParse: idStr: string -> 'ImplId

let inline parseSubjectIdStr<'ImplId when 'ImplId :> AbstractId<'ImplId>>(idStr: string) : 'ImplId =
    'ImplId.TryParse idStr

type MyId = MyId of int
with 
  interface AbstractId<MyId> with
    member this.IdStr =
        match this with | MyId id -> string<int> id
    static member TryParse (idStr: string) =
        System.Int32.Parse idStr |> MyId

let x = MyId 1234
printfn "Id original: %A" x
let idStr = (x :> AbstractId).IdStr
printfn "Id as string: %A" idStr
let xParsed = parseSubjectIdStr<MyId> idStr
printfn "Id parsed from string: %A" xParsed
```

### Expected and actual results

Expected output (I can see it in LINQPad 8, for exapmple):

> Id original: MyId 1234
> Id as string: "1234"
> Id parsed from string: MyId 1234

Actual output (compilation fails):

> Unexpected static interface/override call: Test.AbstractId`1.TryParse (L9,4-L9,26)
> Error: Unexpected static interface/override call: Test.AbstractId`1.TryParse (L9,4-L9,26)
>     at x35 (https://fable.io/repl/js/repl/bundle.min.js:393:73013)
>     at f4n (https://fable.io/repl/js/repl/bundle.min.js:393:79999)
>     at SP (https://fable.io/repl/js/repl/bundle.min.js:393:80630)
>     at https://fable.io/repl/js/repl/bundle.min.js:397:52443
>     at Array.<anonymous> (https://fable.io/repl/js/repl/bundle.min.js:393:81298)
>     at kP (https://fable.io/repl/js/repl/bundle.min.js:393:81186)
>     at Upn.Transform (https://fable.io/repl/js/repl/bundle.min.js:397:36417)
>     at D2n.f (https://fable.io/repl/js/repl/bundle.min.js:397:82281)
>     at https://fable.io/repl/js/repl/bundle.min.js:392:25700
>     at DLe (https://fable.io/repl/js/repl/bundle.min.js:5:350)

### Related information

* Fable version: 4.17.0
* Windows

**Comment by MangelMaxime** [comment]
This is a duplicate of https://github.com/fable-compiler/Fable/issues/3566

According to this issue, if you `inline` the member it will works.

We indeed need to add support for `abstract static` to all Fable languages currently we don't support it yet.

**Comment by DunetsNM** [comment]
hmm, so where exactly do I need to put `inline` in my repro?

**Comment by DunetsNM** [comment]
I see, you suggest to not use static abstract members at all and somehow achieve same behavior with inline functions? It's not very clear what to do from the linked issue

**Comment by ncave** [comment]
Fixed in  #3823
